lighthouse-cli/test/smokehouse/smokehouse.js

```diff
@@ -32,6 +32,8 @@
  * @property {Run[]} runs
  */
 
+import assert from 'assert';
```

Comment:
```suggestion
import {strict as assert} from 'assert';
```

or `import assert from 'assert/strict';`

I believe most of the asserts in this file will be marked `@deprecated` otherwise, though haven't actually verified (and we want strict anyways)